clients/admin-ui/src/app/store.ts

```diff
@@ -5,15 +5,6 @@ import {
   StateFromReducersMapObject,
 } from "@reduxjs/toolkit";
 import { setupListeners } from "@reduxjs/toolkit/query/react";
-import {
```

Comment:
this file has a lot of changes, but they can mostly be broken down into:

* import the slice instead of the reducer or API (and use absolute imports off of `~/feature` so that the imports are grouped together properly)
  * only baseApi needs to be imported since the rest extend off of this one
* remove all the other API calls from the redux-persist blocklist and middleware (again, only baseApi needed here)
* rework the `reducers` dictionary so that they are always indexed by the slice name. this makes things more consistent, since otherwise we have to make sure the string we use is the same as the slice name 
  * i.e. `{ auth: authSlice.reducer }` vs `{[authSlice.name]: authSlice.reducer}`. in this case `authSlice.name` _is_ `auth`, but we had to do just a bit more work and write `auth` as the key. so if we ever changed the name of the slice, we'd have to update in two places. this change makes it so only the one update is needed